### PR TITLE
Add checkstyle

### PR DESCRIPTION
**Pull request by kubik369**

### CODE REVIEW

File: src/main/graphgui/EdgeImplementation.java [hunk 1]
```diff
@@ -8,92 +8,100 @@
...
+  // hrana v opacnom smere
+  private EdgeImplementation reverse;
+  // je tato kopia hrany primarna, t.j. ulozena v zozname hran grafu?
+  private boolean isPrimary;
+
+  public boolean isPrimary() {
+    return isPrimary;
+  }
+
+  @Override public VertexImplementation getOrigin() {
```
Comment (Siegrift) [inline src/main/graphgui/EdgeImplementation.java]: @override on separate line

File: src/main/graphgui/EdgeImplementation.java [hunk 2]
```diff
@@ -8,92 +8,100 @@
...
+    return from;
+  }
+
+  @Override public int getOriginId() {
```
Comment (Siegrift) [inline src/main/graphgui/EdgeImplementation.java]: @override on separate line

File: src/main/graphgui/EdgeImplementation.java [hunk 3]
```diff
@@ -8,92 +8,100 @@
...
+    return getOrigin().getId();
+  }
+
+  @Override public VertexImplementation getDestination() {
```
Comment (Siegrift) [inline src/main/graphgui/EdgeImplementation.java]: @override on separate line

File: src/main/graphgui/EdgeImplementation.java [hunk 4]
```diff
@@ -8,92 +8,100 @@
...
+    return to;
+  }
+
+  @Override public int getDestinationId() {
```
Comment (Siegrift) [inline src/main/graphgui/EdgeImplementation.java]: @override on separate line

File: src/main/graphgui/EdgeImplementation.java [hunk 5]
```diff
@@ -8,92 +8,100 @@
...
+      return reverse;
     }
-
-    @Override public String toString() {
-        Integer ifrom = getOriginId();
-        Integer ito = getDestinationId();
-        return "(" + ifrom + "," + ito + ") value " + value;
+  }
+
+  @Override public boolean isEquivalent(Edge e) {
```
Comment (Siegrift) [inline src/main/graphgui/EdgeImplementation.java]: @override on separate line

File: src/main/graphgui/EdgeImplementation.java [hunk 6]
```diff
@@ -8,92 +8,100 @@
...
+          EdgeImplementation reverse) {
+    this.graph = graph;
+    this.from = from;
+    this.to = to;
+    this.value = defaultValue;
+    this.reverse = reverse;
+    isPrimary = false;
+  }
+
+  @Override public String toString() {
```
Comment (Siegrift) [inline src/main/graphgui/EdgeImplementation.java]: @override on separate line

File: src/main/graphgui/EdgeImplementation.java [hunk 7]
```diff
@@ -8,92 +8,100 @@
...
+    Integer ifrom = getOriginId();
+    Integer ito = getDestinationId();
+    return "(" + ifrom + "," + ito + ") value " + value;
+  }
+
+  @Override public void setValue(int value) {
```
Comment (Siegrift) [inline src/main/graphgui/EdgeImplementation.java]: @override on separate line

File: src/main/graphgui/EdgeImplementation.java [hunk 8]
```diff
@@ -8,92 +8,100 @@
...
+      this.value = value;
+      reverse.value = value;
+      graph.edgeChanged(this);
     }
+  }
 
-    @Override public int getValue() {
-        return value;
-    }
+  @Override public int getValue() {
```
Comment (Siegrift) [inline src/main/graphgui/EdgeImplementation.java]: @override on separate line

File: src/main/graphgui/GraphGUI.java [hunk 9]
```diff
@@ -30,302 +43,279 @@
...
+  private final ExtendedGraph graph = new ExtendedGraph();
+  private final ListView<Edge> listView = new ListView<Edge>(data);
+  private final MyPanel gpanel = new MyPanel(this, graph);
+  private final BorderPane rpanel = new BorderPane();
+  private final GridPane panel = new GridPane();
+  private final BorderPane root = new BorderPane();
+
+  private final Menu menu1 = new Menu("File");
+
+  @Override public void start(Stage primaryStage) {
```
Comment (Siegrift) [inline src/main/graphgui/GraphGUI.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 10]
```diff
@@ -9,133 +10,134 @@
...
+
+  VertexImplementation(GraphImplementation graph, int id, double x, double y) {
+    edges = new ArrayList<EdgeImplementation>();
+    this.id = id;
+    this.graph = graph;
+    this.x = x;
+    this.y = y;
+  }
+
+  @Override public int getId() {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 11]
```diff
@@ -9,133 +10,134 @@
...
+  }
+
+  void internalSetId(int id) {
+    if (id != this.id) {
+      this.id = id;
+      graph.vertexChanged(this);
+    }
+  }
+
+  @Override public Iterable<Vertex> adjVertices() {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 12]
```diff
@@ -9,133 +10,134 @@
...
+    ArrayList<Vertex> a = new ArrayList<Vertex>();
+    for (Edge e : edges) {
+      a.add(e.getDestination());
+    }
+    return Collections.unmodifiableList(a);
+  }
+
+  @Override public Iterable<Integer> adjVertexIds() {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 13]
```diff
@@ -9,133 +10,134 @@
...
+    ArrayList<Integer> a = new ArrayList<Integer>();
+    for (Edge e : edges) {
+      a.add(e.getDestinationId());
+    }
+    return Collections.unmodifiableList(a);
+  }
+
+  @Override public Iterable<Edge> adjEdges() {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 14]
```diff
@@ -9,133 +10,134 @@
...
+
+  void internalAddEdge(EdgeImplementation e) {
+    edges.add(e);
+  }
+
+  boolean internalRemoveEdge(EdgeImplementation e) {
+    return edges.remove(e);
+  }
+
+  @Override public EdgeImplementation findEdge(Vertex v) {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 15]
```diff
@@ -9,133 +10,134 @@
...
+    for (EdgeImplementation e : edges) {
+      if (e.getDestination() == v) {
+        return e;
+      }
+    }
+    return null;
+  }
+
+  @Override public void setValue(int value) {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 16]
```diff
@@ -9,133 +10,134 @@
...
+    if (value != this.value) {
+      this.value = value;
+      graph.vertexChanged(this);
+    }
+  }
+
+  @Override public int getValue() {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 17]
```diff
@@ -9,133 +10,134 @@
...
+    return value;
+  }
+
+  @Override public void setX(double x) {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 18]
```diff
@@ -9,133 +10,134 @@
...
+    if (x != this.x) {
+      this.x = x;
+      graph.vertexChanged(this);
+    }
+  }
+
+  @Override public void setY(double y) {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 19]
```diff
@@ -9,133 +10,134 @@
...
+    if (y != this.y) {
+      this.y = y;
+      graph.vertexChanged(this);
+    }
+  }
+
+  @Override public void setColorName(String colorName) {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 20]
```diff
@@ -9,133 +10,134 @@
...
+    if (!this.colorName.equals(colorName)) {
+      this.colorName = colorName;
+      graph.vertexChanged(this);
+    }
+  }
+
+  @Override public void setSize(double size) {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 21]
```diff
@@ -9,133 +10,134 @@
...
+    if (size != this.size) {
+      this.size = size;
+      graph.vertexChanged(this);
+    }
+  }
+
+  @Override public double getSize() {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 22]
```diff
@@ -9,133 +10,134 @@
...
+    return size;
+  }
 
-    @Override public void setSize(double size) {
-        if(size != this.size) {
-            this.size = size;
-            graph.vertexChanged(this);
-        }
-    }
+  @Override public double getX() {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 23]
```diff
@@ -9,133 +10,134 @@
...
+    return x;
+  }
 
-    @Override public double getSize() {
-        return size;
-    }
+  @Override public double getY() {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

File: src/main/graphgui/VertexImplementation.java [hunk 24]
```diff
@@ -9,133 +10,134 @@
...
-    @Override public String getColorName() {
-        return colorName;
-    }
-
-    @Override public String toString() {
-        Integer id = getId();
-        return id.toString() + " ("
-               + getX() + "," + getY() + ") value " + value;
-    }
+  @Override public String toString() {
```
Comment (Siegrift) [inline src/main/graphgui/VertexImplementation.java]: @override on separate line

**Review by Siegrift — CHANGES_REQUESTED** [review]
I would still prefer running InteliJ autoformatbecause some parts of the code really need it.

File: src/main/graphgui/Editor.java [hunk 25]
```diff
@@ -24,100 +25,97 @@
...
+
+      ok.setOnAction(new EventHandler<ActionEvent>() {
+        @Override
+        public void handle(ActionEvent event) {
+          try {
+            double r = (double)Integer.parseInt(rText.getText());
+            double g = (double)Integer.parseInt(gText.getText());
+            double b = (double)Integer.parseInt(bText.getText());
+            if (r >= 0 && g >= 0 && b >= 0 && r < 256 && g < 256 && b < 256) {
+              vertex.setColorName(new Color(r / 255,g / 255,b / 255,1).toString());
```
Comment (Siegrift) [inline src/main/graphgui/Editor.java]: medzera za 255,

File: src/main/graphgui/Editor.java [hunk 26]
```diff
@@ -24,100 +25,97 @@
...
+              dialog.close();
+            }
+          } catch (Exception e) {
+            System.err.println("Problem s nastavenim farby");
+          }
         }
+      });
+
+      pan.getColumnConstraints().add(new ColumnConstraints(100));
+      pan.setRowIndex(rLabel,0);
```
Comment (Siegrift) [inline src/main/graphgui/Editor.java]: medzery

**Review by Siegrift — CHANGES_REQUESTED** [review]
Este pofixuj medzery a ptm je to ok

File: src/main/graphgui/Editor.java (hunk 25 above)
Comment (kubik369) [reply src/main/graphgui/Editor.java]: Fixed.

File: src/main/graphgui/Editor.java (hunk 26 above)
Comment (kubik369) [reply src/main/graphgui/Editor.java]: Fixed.

**Review by Siegrift — APPROVED** [review]

**Pull request merged**